src/java/org/apache/cassandra/config/Converters.java

```diff
@@ -132,7 +145,7 @@ public Object convert(Object value)
      */
     public Object deconvert(Object value)
```

Comment:
isnt the opposite of convert "unconvert", I see "deconvert" for the first time.

https://www.merriam-webster.com/dictionary/unconvert

"deconvert" is not even there.

Reply:
🤷 it already existed, just fixing the default implementation.